### ISSUE TITLE
Login design issues on mobile devices

### ISSUE THREAD

**Issue by MarcMagdi** [auth0/lock#1783]
### Description

By using the Auth0 Universal Login Page hosted by Auth0, I have 1 issue with the design on some android devices and a second issue on all mobile devices (Android & iPhone)
The design issues include 2 things:

1. The login button is not shown (I thought this was fixed by #1539 but this still exists with me on android devices).
2. When using additional fields to the signup & Use landscape mode (First Name & Last Name per example) the container is broken.

Tested on (Huawei Nova  3i, Galaxy J7 Prime, iPhone 6S, iPhone 7, iPhone 11)
The first design issue doesn't appear on the iPhones I tested with or on Chrome responsive.

### Android Screenshots:
Galaxy J7 Prime
<img src="https://user-images.githubusercontent.com/16214697/72711728-45b4aa80-3b72-11ea-92c5-6d486af70ffd.png" alt="android" width="280"/>

Galaxy J7 Prime
<img src="https://user-images.githubusercontent.com/16214697/72711849-8f04fa00-3b72-11ea-8c5e-6910be9c86a7.png" alt="android" width="620"/>

### iPhone Screenshots:
Login with no issue on iPhone 6S
<img src="https://user-images.githubusercontent.com/16214697/72717097-15720980-3b7c-11ea-8d69-5e1db6e1f505.png" alt="iPhone" width="280"/>

Signup with container issue
iPhone 11
<img src="https://user-images.githubusercontent.com/16214697/72715551-56b4ea00-3b79-11ea-8e8e-8ad2e399053d.png" alt="iPhone" width="620"/>
iPhone 6S
<img src="https://user-images.githubusercontent.com/16214697/72715619-7e0bb700-3b79-11ea-846f-bfc6a9eef32c.png" alt="iPhone" width="620"/>

### Environment

- **Version of this library used:** Lock 11.20.3

**Comment by madine** [comment]
The login button is not shown is logged here too:
https://github.com/auth0/lock/issues/1706

**Comment by stevehobbsdev** [comment]
@MarcMagdi is the login page that exhibits the second issue (container overflow) available somewhere publicly that I can get to?

**Comment by MarcMagdi** [comment]
@stevehobbsdev Here is a test tenant with the bug of the additional fields.
Try it on chrome with a responsive window (rotated window) 
![image](https://user-images.githubusercontent.com/16214697/73132771-3036f900-4028-11ea-880a-8e32a57abe81.png)

Also, I don't see the last merged branch in the releases for the login button issue/

**Comment by stevehobbsdev** [comment]
@MarcMagdi Thanks for the repo, let me get a look at it this week for you.

The release hasn't been done yet, there's another PR that's potentially coming in for that release, just waiting on status.

**Comment by stevehobbsdev** [comment]
@MarcMagdi Could you please verify a fix for me? Find the element with the `auth0-lock-content-wrapper` class on it and add the `overflow-y: hidden` style to it and see if that solves your issue?

With this in place, I get what's in the attached screenshot:

<img width="585" alt="Screenshot 2020-02-04 at 10 34 22" src="https://user-images.githubusercontent.com/766403/73737158-fce81e80-4739-11ea-9a5e-9127323e2c62.png">

**Comment by MarcMagdi** [comment]
@stevehobbsdev This added a new bug that I can't scroll on the form.

**Comment by stevehobbsdev** [comment]
@MarcMagdi 🤔 can you try `overflow-y: scroll` instead - does that fix it for you?

**Comment by MarcMagdi** [comment]
@stevehobbsdev Looks great now!

**Comment by stevehobbsdev** [comment]
@MarcMagdi Ok, but we're getting closer. Let me raise a PR around this.

**Comment by stevehobbsdev** [comment]
@MarcMagdi In what browser are you seeing the scrollbar with `overflow-y: scroll'? I can't reproduce that.

**Comment by MarcMagdi** [comment]
@stevehobbsdev Just updated the repro project with the updated CSS.
I am using Chrome Version 80.0.3987.87 (Official Build) (64-bit) on Linux.

See this as a ref:
https://test-features.auth0.com/login?state=g6Fo2SBCTDB5a3Jxai02VGNQRG9EM0NPc2FMUVp0MS1HejRPcaN0aWTZIDFnZVJNaVlweFFhekR0RjZPM0t0TjdJb25fOTRfT2Nko2NpZNkgYjhiWE40NGZLWWVwRGxpNk1Kd3ZacFVxcVRsaXNHdkU&client=b8bXN44fKYepDli6MJwvZpUqqTlisGvE&protocol=oauth2&prompt=consent&response_type=code&redirect_uri=https%3A%2F%2Fmanage.auth0.com%2Ftester%2Fcallback%3Fconnection%3Dgoogle-oauth2&scope=openid%20profile

**Comment by stevehobbsdev** [comment]
Ok, I've managed to see this now on Chrome, IE11 and Edge on Windows. It's a bit funky because the scrollbar seems to appear and disappear when you transition out of mobile or desktop view.

Try this:

```css
.auth0-lock-content-wrapper {
  overflow-y: scroll;
  scrollbar-width: none;
  -ms-overflow-style: none;
}

.auth0-lock-content-wrapper::-webkit-scrollbar {
  width: 0;
  height: 0;
}
```

Where `.container` is the selector

**Comment by MarcMagdi** [comment]
@stevehobbsdev Looks good on my end! 
Also, I'd prefer the scroll bar mentioned on #1800 with the old version as an indicator there is still content. 
But for now, looks great.

**Comment by stevehobbsdev** [comment]
This will perhaps need some more thought then. Unfortunately at this point we cannot have both.

The `overflow` style on that element is there to help display the login button correctly at the bottom of the screen. I've identified #1758 as the change which broke the container element, but reversing that still produces scrollbars, so we end up in the same position.

Let me raise the PR and put all the trade-offs in there, perhaps we can come to a solution.
